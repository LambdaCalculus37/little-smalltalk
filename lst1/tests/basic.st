Class Main
[
	main
		88 print.
		3.14159 print.
		'this is it' print.
		#(this is also it) print.
		88 respondsTo: #+ ; print.
		Object respondsTo.
		smalltalk at: 3 put: #(22 17).
		(smalltalk at: 3) print.
		Smalltalk respondsTo.
]
