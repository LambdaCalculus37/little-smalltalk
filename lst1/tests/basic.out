Little Smalltalk
	88
3.14159
this is it
#( #this #is #also #it )
True
shallowCopy
respondsTo:
printString
print
notNil
next
isNil
isMemberOf:
isKindOf:
first
error:
do:
deepCopy
copy
class
asSymbol
asString
~=
=
~~
==
#( 22 17 )
time:
sh:
perform:withArguments:
noDisplay
doPrimitive:withArguments:
displayAssign
display
debug:
date
clearScreen
Main
	

